contracts/nft/mainnet/NFTMainnetBridgingToPolygonFacet.sol

```diff
@@ -120,10 +120,37 @@ contract NFTMainnetBridgingToPolygonFacet {
         __depositFor(tokenIds_, amounts_);
     }
 
-    function bridgeNFTsV2(
-        uint256[] calldata tokenIds,
-        uint256[] calldata amounts
-    ) external {
-        // TODO: waiting for code Syed is working on
+    /**
+     * @notice Gets all of our NFTs (staked and unstaked) migrates to ERC1155 (if ERC721), then
+     *  bridges to polygon
+     * @param tokenIds The tokenIds that we are sending. First array is the ERC721, if any. Second array
+     *  are our ERC-1155 tokenIDs
+     */
+
+    function bridgeNFTsV2(uint256[] calldata tokenIds) external {
+        EnumerableSet.UintSet storage stakedNFTs = NFTLib.s().stakedNFTs[
+            msg.sender
+        ];
+
+        // we are creating a new array so that we can overrwrite each element
+        // with a newTokenId.
+        uint256[] memory tokenIds_ = new uint256[](tokenIds.length);
```

Comment:
No need for this array since the IDs are already being passed in as a parameter. Also doesn't seem like we are ever setting values on this variable.